standardization/src/test/scala/za/co/absa/enceladus/standardization/interpreter/StandardizationInterpreter_DecimalSuite.scala

```diff
@@ -131,5 +132,93 @@ class StandardizationInterpreter_DecimalSuite extends FunSuite with SparkTestBas
     val std = StandardizationInterpreter.standardize(src, desiredSchema, "").cache()
     logDataFrameContent(std)
 
-    assertResult(exp)(std.as[DecimalRow].collect().sortBy(_.description).toList)  }
+    assertResult(exp)(std.as[DecimalRow].collect().sortBy(_.description).toList)
+  }
+
+  test("No pattern, but altered symbols") {
+    val input = Seq(
+      ("01-Normal", "123:456"),
+      ("02-Null", null),
+      ("03-Far negative", "N100000000:999"),
+      ("04-Wrong", "hello"),
+      ("05-Not adhering to pattern", "123456.789")
+    )
+
+    val decimalSeparator = ":"
+    val minusSign = "N"
```

Comment:
Interesting idea. Actually it would work, because:
* Non-default `Radix` (<> 10) beats any `Pattern` (`Pattern` would be ignored then, and validation gives _Warning_ about it)
* If pattern is not used, the defined minus sign is replaced with "real" minus sign. And then parsed.
* If there are no collisions (and in your example there are not), it all works marvelously.
Well question arises what if there are collisions? Then the result will be hard to predict. It's not just this example. But what for example, what if _decimal separator_ is defined as _E_.
I considered to include this in the Validation. But if to be done properly it deserves it's whole ticket. It's not small task.

Reply:
Please add test for the radix 5 for example

Reply:
Will add, but to Integral parsing. Only whole numbers support `Radix`